internal/operators/cnv/cnv_operator.go

```diff
@@ -160,3 +150,20 @@ func (o *operator) GetProperties() models.OperatorProperties {
 func (o *operator) GetMonitoredOperator() *models.MonitoredOperator {
 	return &Operator
 }
+
+// GetHostRequirementsForRole provides operator's requirements towards host in a given role
+func (o *operator) GetHostRequirementsForRole(_ context.Context, _ *common.Cluster, role models.HostRole) (*models.ClusterHostRequirementsDetails, error) {
+	switch role {
+	case models.HostRoleMaster:
+		return &models.ClusterHostRequirementsDetails{
+			CPUCores: masterCPU,
+			RAMMib:   masterMemory,
```

Comment:
What is the memory unit we expect here? Is it in bytes?

Reply:
MiB